### PR TITLE
Multi comparison line graphs

### PR DESCRIPTION
**Pull request by ChelsiAlise**
# Conflicts:
#	static/index.js

### CODE REVIEW

File: static/index.js [hunk 1]
```diff
@@ -170,21 +172,115 @@ function makeDogPoints(dogs, setPointKeysFunc) {
...
 // chart 1 - Awake Versus Rest of All Dogs by Name
 function createChartOne() {
     // create series
-    var series_data = makeDogPoints(filtered_dogs, function(point) {
+    var series_data = makeDogPoints(filtered_dogs, function (point) {
         point.x = point.awake / point.total * 100,
-        point.y = point.rest / point.total * 100,
-        point.marker = {radius: Math.pow(point.total / filtered_dogs_max_total, .2) * 6};
+            point.y = point.rest / point.total * 100,
+            point.marker = { radius: Math.pow(point.total / filtered_dogs_max_total, .2) * 6 };
```
Comment (BenTheElder) [inline static/index.js]: is there a reason these lines are indented an extra level?

File: static/index.js [hunk 2]
```diff
@@ -411,11 +513,11 @@ function createChartThree() {
 // chart 4 Activity Percentage of All Dogs
 function createChartFour() {
     // create the series
-    var series_data = makeDogPoints(filtered_dogs, function(point) {
+    var series_data = makeDogPoints(filtered_dogs, function (point) {
         point.y = point.active / point.total * 100,
-        point.marker = {radius: Math.pow(point.total / filtered_dogs_max_total, .2) * 4};
+            point.marker = { radius: Math.pow(point.total / filtered_dogs_max_total, .2) * 4 };
```
Comment (BenTheElder) [inline static/index.js]: same here, shouldn't this line up with the line above? This looks like a pointless change. (perhaps by an auto-formatter?)

Edit: same for the matching line in the other charts below.

**Comment by BenTheElder** [comment]
This needs to be tagged with a milestone.

File: static/index.js (hunk 2 above)
Comment (ChelsiAlise) [inline static/index.js]: its probably just from me using the auto format in visual studio code. Doesn't change any functionality. code still works

Comment (BenTheElder) [inline static/index.js]: right, but it introduces merge conflicts when lines are changed, and it reads poorly when the indent doesn't match.

just a nit.

**Comment by BenTheElder** [comment]
Some Comments:
 - no year is shown for the dates.
 - minutes should probably be labeled `mins` in the formatter and as integers for consistency
 - the line appears to be fitted, if so the fit method should probably be listed somewhere.

**Comment by ChelsiAlise** [comment]
I'm not sure what you mean by the following comments:

- "Edit: we may also want to consider showing % of total for the day instead of raw totals. Either way the graphs are skewed by being in the 70% <= total <= 100% range."
- "the line appears to be fitted, if so the fit method should probably be listed somewhere."

**Comment by ChelsiAlise** [comment]
no year is shown for the dates. **NOT SURE WHY THIS IS OCCURING. THIS ISSUE CAN BE FIXED IN A LATER SPRINT**
minutes should probably be labeled mins in the formatter and as integers for consistency **FIXED**

**Comment by BenTheElder** [comment]
By the first I meant that the graph also fluctuates by amount of data. If we scaled each value as a percent of the total for the day it might have less variance. I think they're more interested in ratios of time spent sleeping etc rather than the absolute values.

For the second point I meant that since the lines appear to be curved instead of straight lines from point to point, that implies some sort of line fit (EG a n-order polynomial), if that's true then we should note the fit method somewhere so that when the graphs are exported the user knows where those curved came from.


The year isn't showing intentionally according the comments in the formatter, which appear to be copied from the demo for this type of graph. [The docs for formatting](http://www.highcharts.com/docs/chart-concepts/labels-and-string-formatting) show that `%Y` is used to format a year.

**Comment by BenTheElder** [comment]
Closing this due to history conflicts after re-purging json blobs for security, we should move this to a fresh PR

**Pull request closed without merging**